include/types/wlr_output_layout.h

```diff
@@ -0,0 +1,32 @@
+#ifndef TYPES_WLR_OUTPUT_LAYOUT_H
+#define TYPES_WLR_OUTPUT_LAYOUT_H
+
+#include <wlr/types/wlr_output_layout.h>
+
+struct wlr_output_layout_output_state {
```

Comment:
Any reason to keep this around?

Reply:
No, I think the whole point of this state struct was to keep some stuff private.